src/helpers/figures/charts/chart_common.ts

```diff
@@ -272,6 +276,46 @@ export function toExcelDataset(getters: CoreGetters, ds: DataSet): ExcelChartDat
     };
   }
 
+  let trend: ExcelChartTrendConfiguration | undefined = undefined;
+  if (ds?.trend?.type) {
+    trend = {
+      type: CHART_TRENDLINE_TYPE_CONVERSION_MAP_REVERSE[ds.trend.type],
+    };
+    if (ds.trend.color) {
+      trend = {
+        ...trend,
+        color: ds.trend.color,
+      };
+    }
+    if (ds.trend.type === "polynomial" && ds.trend.order) {
+      if (ds.trend.order === 1) {
+        trend = {
+          ...trend,
+          type: "linear",
+        };
+      } else {
+        trend = {
+          ...trend,
+          order: toExcelPolynomialTrendlineOrder(ds.trend.order),
```

Comment:
A function only used once doesn't seems really usefull. Maybe something like `Math.max(ds.trend.order, 6)` ? Or even with a const `MAX_EXCEL_POLYNOMIAL_DEGREE` as 6 if you want to make is clearer ?

Reply:
Fair enough!